codechecker_lib/analyzers/analyzer_types.py

```diff
@@ -313,7 +314,8 @@ def construct_result_handler(args,
                              severity_map,
                              skiplist_handler,
                              lock,
-                             store_to_db=False):
+                             store_to_db=False,
+                             export_plist_path=None):
```

Comment:
Couldn't we solve this somehow else? There are already too many arguments for this function.
Maybe splitting up result handler construction would be better.
`run_id, severity_map, store_to_db` are not required for a file output only result handler.